fuzz/fuzz_targets/failing.rs

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(#3503): This target is temporary. Remove the file once we have enough data.
+//! This is an intentionally failing fuzz target for testing.
+
+#![no_main]
+#![feature(async_closure)]
+
+use libfuzzer_sys::fuzz_target;
+use oak_remote_attestation::crypto::get_sha256;
```

Comment:
This dep seems unnecessary, maybe something simpler like https://doc.rust-lang.org/std/hash/trait.Hasher.html would be enough. Anyways feel free to go ahead with this and then remove later as noted by your TODOs.

Reply:
I intentionally wanted to fuzz something in the code base, that is why I added this dependency :)
Since this will be removed soon anyway I'll keep it like this.